searchlib/src/vespa/searchlib/common/bitvector.cpp

```diff
@@ -351,8 +351,8 @@ BitVector::create(Index start, Index end)
 BitVector::UP
 BitVector::create(const BitVector & org, Index start, Index end)
 {
-    return (start == 0)
-           ? create(end)
+    return ((start == 0) && (end == org.size()))
```

Comment:
Consider also checking that org.getStartIndex() is zero or adjusting constructor for AllocatedBitVector to handle
that case.

Reply:
Fixed using first option.